File: PB22_dashboard.py
from flask import Flask, render_template, jsonify

app = Flask(__name__)

# Datos simulados (pueden venir de una base de datos en el futuro)
data = {"Pendiente": 5, "Aprobada": 10, "Rechazada": 3, "Anulada": 2}

@app.route("/")
def home():
    return render_template("dashboard.html", data=data)

@app.route("/data")
def get_data():
    return jsonify(data)

if __name__ == "__main__":
    app.run(debug=True)

File: PB22_dashboard.js
const express = require('express');
const app = express();

app.set('view engine', 'ejs');
app.use(express.static('public'));

const data = { Pendiente: 5, Aprobada: 10, Rechazada: 3, Anulada: 1 };

app.get('/', (req, res) => {
  res.render('dashboard', { data });
});

app.listen(3000, () => {
  console.log('Servidor ejecutándose en http://localhost:3000');
});
